README.md

```diff
@@ -65,6 +65,12 @@ sensible defaults.
     # Useful for jobs where the matrix is additive e.g. additional Cargo features.
     # default: "true"
     save-if: ""
+
+    # Determines whether to use github or buildjet cache
+    # If `true`, will use buildjet for saving cached information.
+    # Useful when using self-hosted runners, esepcially if using buildjet runners.
+    # default: "false"
+    use-buildjet: ""
```

Comment:
Maybe a more generic `cache-backend` option that defaults to `"github"` is preferable in case this will ever be expanded to even more backends.